pallets/eth-bridge/src/lib.rs

```diff
@@ -556,6 +549,69 @@ pub mod pallet {
 
             Ok(().into())
         }
+
+        #[pallet::call_index(7)]
+        #[pallet::weight(<T as Config>::WeightInfo::add_eth_tx_hash())]
+        pub fn submit_latest_ethereum_block(
+            origin: OriginFor<T>,
+            author: Author<T>,
+            latest_seen_block: u32,
+            _signature: <T::AuthorityId as RuntimeAppPublic>::Signature,
+        ) -> DispatchResultWithPostInfo {
+            ensure_none(origin)?;
+            ensure!(Self::active_ethereum_range().is_some(), Error::<T>::VotingEnded);
+            ensure!(
+                author_has_cast_event_vote::<T>(&author.account_id) == false,
+                Error::<T>::EventVoteExists
+            );
+
+            let nominated_range =
+                events_helpers::compute_finalised_block_range_for_latest_ethereum_block(
+                    latest_seen_block,
+                );
+            if let Some(events_partition) =
+                events_helpers::discovered_eth_events_partition_factory(nominated_range, Vec::new())
```

Comment:
Why are we calling this function with an empty array?

Reply:
its a factory method that produces partitions given an input of events. In the case of the initial range consensus we are interested in producing a partition with no events and the nominated_range